<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LockPage</class>
 <widget class="QWidget" name="LockPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI Light</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="miniBtn">
   <property name="geometry">
    <rect>
     <x>617</x>
     <y>16</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>323</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,34,76);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="enterBtn">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>350</y>
     <width>135</width>
     <height>34</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>78</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_version">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>114</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px;</string>
   </property>
   <property name="text">
    <string>v1.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdLineEdit">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>280</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>647</x>
     <y>16</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="syncLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>445</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: white;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
